zeppelin-web/src/app/notebook/notebook-actionBar.html

```diff
@@ -31,10 +31,20 @@ <h3>
       <button type="button"
               class="btn btn-default btn-xs"
               ng-click="runAllParagraphs(note.id)"
-              ng-class="{'disabled':isNoteRunning()}"
+              ng-if=!isNoteRunning()
               tooltip-placement="bottom" uib-tooltip="Run all paragraphs"
               ng-disabled="revisionView">
         <i class="icon-control-play"></i>
+      </button>
+        <button type="button"
+              class="btn btn-default btn-xs"
+              ng-click=stopNoteExecution(note.id)
+              style="color: red"
+              ng-if=isNoteRunning()
+              tooltip-placement="bottom"
+              uib-tooltip="Stop execution"
+              ng-disabled="revisionView">
+        <i class="icon-control-pause"></i>
```

Comment:
that's a pause icon - is there a stop icon?

Reply:
simple-line-icons.css not contains 'stop' button. Therefore, I used 'icon-control-pause'. 
In paragraph-control.html 'icon-control-pause' is also used to stop a paragraph.